frontend/cypress/integration/login/survey-list/survey-list.spec.js

```diff
@@ -0,0 +1,33 @@
+context('survey list', ()=> {
+  const login = {
+    email:'student@gmail.com',
+    password:'123456'
+  };
+
+  before(() => {
+    cy.visit(`${Cypress.config("baseUrl")}/login`)
+  });
+
+  it('Should open survey page', ()=>{
+    cy.globalLogin(login);
```

Comment:
Usually, the login data is not relevant, so it probably can be added to support like https://github.com/edimossilva/memory_app/blob/master/e2e/cypress/support/commands.js#L13